### ISSUE TITLE
UI randomly goes blank

### ISSUE THREAD

**Issue by Ricket** [aidancbrady/Mekanism#4170]
When looking at machine UIs, they randomly go empty and then you have to escape and right click back into it to see the items again. This happens quite frequently.

Version: Built from latest 1.10 branch as of Jan 29 2017 (7e3c806a8f136526129967ba1b25946d8f05ccd9). Added to direwolf20-1.10 pack (version 1.4.1).

**Comment by thommy101** [comment]
Do you have a screenshot of it happening? That would help a lot with trying to solve it.

**Comment by Ricket** [comment]
Sure. I believe this happens with all the machines, but take the metallurgic infuser for example. Best if you don't add speed upgrades. I put in 4 compressed redstone:

![mekanism_beforeinfusing](https://cloud.githubusercontent.com/assets/196978/22407182/992aa516-e62f-11e6-9437-3b96f99ca3a7.png)

Then I put in 32 iron ingots:

![mekanism_infusing32ingots](https://cloud.githubusercontent.com/assets/196978/22407187/b9cd41c0-e62f-11e6-8776-ec5a90286147.png)

If you just sit there and watch it, the UI will randomly blank the items, the redstone and RF meters remain filled and the arrow does continue the filling animation:

![mekanism_uiblanked](https://cloud.githubusercontent.com/assets/196978/22407199/e573e072-e62f-11e6-93c0-81727b5ac1f1.png)

Once that one item finishes processing and the next one begins, the items reappear (until they randomly disappear again) -- I guess because the UI updates the numbers of items in the slots.

And then once all the processing finishes, you can just stare at the completed items and they will eventually disappear (and then you have to escape and re-enter the UI to see them and pull them out). This is a screenshot of after completion, there are 32 enriched alloys in the machine and I just waited (did nothing) for about 10-15 seconds after the crafting completed and they disappeared:

![mekanism_afterfinishedtheydisappeared](https://cloud.githubusercontent.com/assets/196978/22407207/0d05c632-e630-11e6-8a1c-d63f785d5a5e.png)

ESC and then right-click the machine again:

![mekanism_afterfinished_closereopen](https://cloud.githubusercontent.com/assets/196978/22407213/2bbac546-e630-11e6-8c7c-1a7545a2c43f.png)

**Comment by jburrows82** [comment]
Are you using Optifine by chance?

**Comment by Ricket** [comment]
I just disabled optifine and confirmed that the bug still happens without optifine. On average, every two or three ingots, the UI goes blank til the next ingot refreshes it.

However, there is something more to reproducing it and I'm not sure what that is yet... I teleported to the end and set up just a metallurgic infuser and ultimate energy cell, and tried the above and the UI did not go blank at all. I switched to my single player test world and I have tried setting up various machines that are in my base and have not been able to reproduce the UI bug. But every time I try it in my base in the multiplayer server, it happens within just a couple ingots.

I can do some more intensive trial and error later this week. Maybe download the map from the server, delete chunks via MCedit and see if I can narrow down how to reproduce this issue. Any other ideas?

Could this be at all related to the #3977 "Machines rotating by themselves"? I have also been unable to reproduce that in my single player world so far, and it also happens ALL THE TIME in my base on the server. Also happens to the other player on my server, whose base is over 1000 blocks away from mine; and he also sees this UI bug.
